sunpy/coordinates/tests/test_spice.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import spiceypy
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, frame_transform_graph
+from astropy.tests.helper import assert_quantity_allclose
+from astropy.utils.data import download_file
+
+from sunpy.coordinates import spice
+
+
+@pytest.mark.remote_data
+def setup_module():
```

Comment:
can this not be a pytest fixture? I think you can have tests which use a fixture inherit the marks on the fixture one way or another.

You could also make this a module scoped fixture so it is only called once.

Reply:
I think this should be a fixture, even if we have to mark all the tests explicitly.

Reply:
Done